core/DataAccess/Model.php

```diff
@@ -867,4 +867,19 @@ public function releaseInProgressInvalidation($idinvalidation)
         $sql = "UPDATE $table SET status = " . ArchiveInvalidator::INVALIDATION_STATUS_QUEUED . " WHERE idinvalidation = ?";
         Db::query($sql, [$idinvalidation]);
     }
+
+    public function resetFailedArchivingJobs()
+    {
+        $table = Common::prefixTable('archive_invalidations');
+        $sql = "UPDATE $table SET status = ? WHERE status = ? AND ts_invalidated < ?";
```

Comment:
@diosmosis could we add a test for this? and might be also good to document this in https://developer.matomo.org/guides/archiving ?